ansible/mdadm.yml

```diff
@@ -0,0 +1,18 @@
+- hosts: cluster_mdadm
+  vars:
+    mdadm_arrays: "{{ mdadm_arrays }}"
+  roles:
+  - { role: mrlesmithjr.mdadm, become: yes }
+  tasks:
+  # NOTE (brtknr): The mdadm role currently mounts the multidisk mountpoint by
+  # default. We do not want this behaviour as the subsequent roles will decide
+  # where to mount this.
```

Comment:
Does that make this playbook not idempotent?

Reply:
It only unmounts whats mounted by mdadm role so it is idempotent. There does not seem to be an option to not mount the device by default. This is a workaround which mounts the device to a dummy location and unmounts immediately.

Reply:
It won't be a problem to mount the device twice if it has already been mounted elsewhere? Could we look at making the mount optional upstream in the mdadm role?

Reply:
Thats another option!